simue/simue.go

```diff
@@ -200,6 +201,10 @@ func ImsiStateMachine(profile *profctx.Profile, pCtx *profctx.ProfileUeContext,
 			proc_fail = true
 		case msg := <-pCtx.ReadChan:
 			pCtx.Log.Infoln("imsiStateMachine received result ")
+			end_ts := time.Now()
+			diff := end_ts.Sub(start_ts)
+			pCtx.Log.Infof("[LOG][E2E], %v, %v, %v\n", msg.Supi, diff.Milliseconds(), msg.GetEventType().String())
```

Comment:
`pCtx` already provides the supi. So, there is no need to add `msg.Supi` in the log. I think this should be something like this:
```suggestion
			pCtx.Log.Infof("[E2E], latency: %v, %s\n", diff.Milliseconds(), msg.GetEventType().String())
```

Reply:
FYI, this is how the output looks like with your changes:
```
2024-03-15T15:15:41Z [INFO][GNBSIM][ProfUeCtx][imsi-208930100007561] [LOG][E2E], imsi-208930100007561, 105, PROC-PASS-EVENT
```

My suggestion outputs the following:
```
2024-03-15T15:42:20Z [INFO][GNBSIM][ProfUeCtx][imsi-208930100007564] procedure: UE-REQUESTED-PDU-SESSION-RELEASE-PROCEDURE, status: PROC-PASS-EVENT, E2E latency [ms]: 256
```

Reply:
FYI, this is how the output looks like with your changes:
```
2024-03-15T15:15:41Z [INFO][GNBSIM][ProfUeCtx][imsi-208930100007561] [LOG][E2E], imsi-208930100007561, 105, PROC-PASS-EVENT
```

My suggestion outputs the following:
```
2024-03-15T15:42:20Z [INFO][GNBSIM][ProfUeCtx][imsi-208930100007564] procedure: UE-REQUESTED-PDU-SESSION-RELEASE-PROCEDURE, status: PROC-PASS-EVENT, E2E latency [ms]: 256
```

Reply:
I agree with the proposed format but prefer 'latency' over `RTT`.